sklearn/metrics/tests/test_common.py

```diff
@@ -464,8 +484,17 @@ def test_symmetry():
     # Symmetric metric
     for name in SYMMETRIC_METRICS:
         metric = ALL_METRICS[name]
-        assert_allclose(metric(y_true, y_pred), metric(y_pred, y_true),
-                        err_msg="%s is not symmetric" % name)
+        if name in METRIC_UNDEFINED_BINARY:
```

Comment:
My suggestion is to avoid modifying common tests, unless tests are failing.

Reply:
I disagree. Until now we did not have a symmetric metric with binary undefined. The common tests are not complete without this change.